index.ts

```diff
@@ -13,7 +13,9 @@ export enum LabelStatus {
   IGNORED_CODE_NAC_INCONNU = 'ignored_codeNACInconnu',
   IGNORED_CARACTERE_INCONNU = 'ignored_caractereInconnu',
   IGNORED_DATE_AVANT_MISE_EN_SERVICE = 'ignored_dateAvantMiseEnService',
-  IGNORED_CONTROLE_REQUIS = 'ignored_controleRequis'
+  IGNORED_CONTROLE_REQUIS = 'ignored_controleRequis',
+  IGNORED_DECISION_NON_PUBLIQUE_PAR_ZONAGE = 'ignored_decisionNonPubliqueParZonage',
+  IGNORED_CODE_NAC_DECISION_PARTIELLEMENT_PUBLIQUE_PAR_ZONAGE = 'ignored_codeNACdeDecisionPartiellementPubliqueParZonage'
```

Comment:
Ma doc n'étais pas très juste, il faut plutôt mettre 
```suggestion
  IGNORED_DECISION_PARTIELLEMENT_PUBLIQUE_PAR_ZONAGE = 'ignored_decisionPartiellementPubliqueParZonage'
```